src/system/MailerModule/Form/Type/ConfigType.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * This file is part of the Zikula package.
+ *
+ * Copyright Zikula Foundation - http://zikula.org/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zikula\MailerModule\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Zikula\Bundle\CoreBundle\DynamicConfigDumper;
+use Zikula\Common\Translator\TranslatorInterface;
+use Zikula\Common\Translator\TranslatorTrait;
+use Zikula\ExtensionsModule\Api\VariableApi;
+
+/**
+ * Configuration form type class.
+ */
+class ConfigType extends AbstractType
+{
+    use TranslatorTrait;
+
+    /**
+     * @var array
+     */
+    protected $dataValues;
+
+    /**
+     * ConfigType constructor.
+     *
+     * @param TranslatorInterface $translator   Translator service instance.
+     * @param DynamicConfigDumper $configDumper DynamicConfigDumper service instance.
+     * @param VariableApi         $variableApi  VariableApi service instance.
+     */
+    public function __construct(TranslatorInterface $translator, DynamicConfigDumper $configDumper, VariableApi $variableApi)
+    {
+        $this->setTranslator($translator);
+
+        $params = $configDumper->getConfiguration('swiftmailer');
```

Comment:
can we just inject the config params without loading the whole dumper? (unless that isn't what is needed).